prometheus-to-sd/translator/metric_descriptor_cache_test.go

```diff
@@ -17,9 +17,13 @@ limitations under the License.
 package translator
 
 import (
+	"fmt"
 	"testing"
 
+	"github.com/GoogleCloudPlatform/k8s-stackdriver/prometheus-to-sd/config"
```

Comment:
We usually separate project-local imports from the rest. I.e.

```
  dto "github.com/prometheus/client_model/go"
  "github.com/stretchr/testify/assert"
  v3 "google.golang.org/api/monitoring/v3"

  "github.com/GoogleCloudPlatform/k8s-stackdriver/prometheus-to-sd/config"
```

Reply:
Done, thanks.